internal/proto/controller/api/resources/targets/v1/target.proto

```diff
@@ -334,3 +334,8 @@ message SshPrivateKeyCredential {
   // The optional passphrase of the private_key
   string private_key_passphrase = 3; // @gotags: `class:"secret"`
 }
+
+// The layout of the struct for "credential" field in SessionCredential for a kv credential type.
+message KVCredential {
+  google.protobuf.Struct secret = 1; // @gotags: `class:"secret"`
```

Comment:
The pluralization here does not match the credential service. Here it's singular `secret`, there it's plural `secrets`